dd-sdk-android/src/test/kotlin/com/datadog/android/utils/assertj/DeserializedActionEventAssert.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.utils.assertj
+
+import com.datadog.android.rum.model.ActionEvent
+import org.assertj.core.api.AbstractAssert
+import org.assertj.core.api.Assertions.assertThat
+
+internal class DeserializedActionEventAssert(actual: ActionEvent) :
+    AbstractAssert<DeserializedActionEventAssert, ActionEvent>(
+        actual,
+        DeserializedActionEventAssert::class.java
+    ) {
+
+    fun isEqualTo(expected: ActionEvent): DeserializedActionEventAssert {
+        assertThat(expected)
```

Comment:
```suggestion
        assertThat(actual)
```

Same for other `DeserializedXXX` classes

Reply:
Ouch...thanks for seeing that.